### PR TITLE
GH-19: add build order for archers

### PR DESCRIPTION
**Pull request by LaymanTruant**

### CODE REVIEW

File: app/content/build/archer.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+---
+title: "小弓開局"
+---
+
+## 簡介
+
+封建時代的弓兵，俗稱小弓。小弓開局是非常經典的起手開局，也是目前最主流的開局技巧之ㄧ，和另一肉馬快攻相比的優勢在於生產小弓不需要花費食物，讓資源更能有效利用，不會衝突村民的生產，更受高端玩家的喜愛。一兩隻小弓並無威脅，但一隊小弓就很具威力。一隊8隻小弓只要射2箭就能殺死一隻斥侯。小弓是遠程單位，對手無法使用快速牆阻擋，因為小弓可以隔著牆射到村民。小弓雖然強大，但成形慢，所以往往會先被肉馬快攻一波，等到小弓數量起來才有辦法反擊對手。在封建時代可以有效阻止小弓的單位只有鷹斥侯和矛兵，另外在家裡插瞭望箭塔也可以有效防禦。
```
Comment (milkoya) [inline app/content/build/archer.md:7]:
```suggestion
封建時代的弓兵，俗稱小弓。小弓開局是非常經典的起手開局，也是目前最主流的開局技巧之一，和另一肉馬快攻相比的優勢在於生產小弓不需要花費食物，讓資源更能有效利用，不會衝突村民的生產，更受高端玩家的喜愛。一兩隻小弓並無威脅，但一隊小弓就很具威力。一隊8隻小弓只要射2箭就能殺死一隻斥侯。小弓是遠程單位，對手無法使用快速牆阻擋，因為小弓可以隔著牆射到村民。小弓雖然強大，但成形慢，所以往往會先被肉馬快攻一波，等到小弓數量起來才有辦法反擊對手。在封建時代可以有效阻止小弓的單位只有鷹斥侯和矛兵，另外在家裡插瞭望箭塔也可以有效防禦。
```

Comment (LaymanTruant) [reply app/content/build/archer.md]: Thanks.

File: app/content/build/archer.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+## 應對策略
+
+小弓走到敵人家可能有幾種狀況：
+1. 敵人家裡圍死：嘗試尋找破口，如果沒有破口可以直接射木牆，如果對方村民在木牆後方補蓋房舍，可以隔牆直接射村民阻止。
+1. 肉馬：用小弓拉打，肉馬很難和一隊小弓抗衡，但如果小弓量不足的話，儘量在被殲滅前對敵方單位造成傷害。
+1. 矛兵防守：少量的矛兵，或是對手尚未研發弓兵科技的時候可以硬吃。如果對手產太多升級過的矛兵，這是錯誤的，因為會消耗過多資源(食物)導致拖累升級進度，可以轉兵產肉馬或是先退回家升級城堡時代。
+1. 瞭望箭塔：箭塔無法移動，可尋找其他資源點攻擊。高端玩家有時會將弓兵移動至箭塔下方，因為封建時代並無城垛，在箭塔下方弓兵不會受到傷害。
+1. 小弓互射：量多者勝，可移動至上風增加攻擊力，也可以適度地生產一些矛兵對抗。
+1. 鷹勇士：鷹勇士是小弓剋星，但在封建時代鷹勇士生產時間很長，不易量產。一兩隻鷹勇士仍能靠一隊小弓獲勝，但如果鷹勇士數量增加，需要考慮轉兵或是撤退升級城堡時代。
```
Comment (milkoya) [inline app/content/build/archer.md:36]:
```suggestion
1. 敵人家裡圍死：嘗試尋找破口，如果沒有破口可以直接射木牆，如果對方村民在木牆後方補蓋房舍，可以隔牆直接射村民阻止。
2. 肉馬：用小弓拉打，肉馬很難和一隊小弓抗衡，但如果小弓量不足的話，儘量在被殲滅前對敵方單位造成傷害。
3. 矛兵防守：少量的矛兵，或是對手尚未研發弓兵科技的時候可以硬吃。如果對手產太多升級過的矛兵，這是錯誤的，因為會消耗過多資源(食物)導致拖累升級進度，可以轉兵產肉馬或是先退回家升級城堡時代。
4. 瞭望箭塔：箭塔無法移動，可尋找其他資源點攻擊。高端玩家有時會將弓兵移動至箭塔下方，因為封建時代並無城垛，在箭塔下方弓兵不會受到傷害。
5. 小弓互射：量多者勝，可移動至上風增加攻擊力，也可以適度地生產一些矛兵對抗。
6. 鷹勇士：鷹勇士是小弓剋星，但在封建時代鷹勇士生產時間很長，不易量產。一兩隻鷹勇士仍能靠一隊小弓獲勝，但如果鷹勇士數量增加，需要考慮轉兵或是撤退升級城堡時代。
```
Is this index supposed to be 1?

Comment (LaymanTruant) [reply app/content/build/archer.md:36]: It's irrelevant in markdown, I'd like to use `1.`, please refer to [Ordered Lists](https://www.markdownguide.org/basic-syntax/#ordered-lists).

Comment (milkoya) [reply app/content/build/archer.md:36]: Sure, no problem

**Pull request merged**